package.json

```diff
@@ -39,7 +39,7 @@
     "@edx/frontend-component-footer": "10.0.3",
     "@edx/frontend-component-header": "2.0.3",
     "@edx/frontend-platform": "1.1.9",
-    "@edx/paragon": "7.2.1",
+    "@edx/paragon": "^12.3.0",
```

Comment:
The nearby convention seems to be pegging specific versions- anything to note about allowing later patch versions for paragon? (Probably because we expect more updates as the branding solidifies, but wanted to check how we decide)

Reply:
Good call, will peg to specific 12.3.0

Reply:
With checkout having a lot of visibility, I wouldn't want Paragon to be at a later version when you run `npm install`, which with the `^` will do that from what I understand, and that prefix allows more than just bug fixes. I don't think using `^` would allow major release changes (like for version 13 or something) but good to keep it at a specific version and not increase the version range it would allow. The caret `^` symbol is the default when you install a package with --save, that's how it ended up there

Reply:
that makes sense then (to change to 12.3.0)- thanks!